libs/barista-components/radio/src/radio-group.ts

```diff
@@ -113,11 +119,40 @@ export class DtRadioGroup<T> implements AfterContentInit, CanDisable {
     this._markRadiosForCheck();
   }
 
+  /** Unique id of the element. */
+  @Input()
+  get id(): string {
+    return this._id;
+  }
+  set id(value: string) {
+    this._id = value || this._uid;
+  }
+  private _uid = `dt-checkbox-${nextUniqueId++}`;
+  private _id: string;
```

Comment:
If the setter is never called, the id will never be set. We need to add a default here.

Reply:
shouldn`t be the default value for the _id? Because if you never call the setter the id will be undefined.

```diff
 @Input()
  get id(): string {
    return this._id;
  }
  set id(value: string) {
-    this._id = value || this._uid;
+    this._id = value;
  }
  private _uid = `dt-checkbox-${nextUniqueId++}`;
- private _id: string;
+ private _id: string = this._uid;
```

Reply:
Setter is called in the constructor. It now works the same as in the checkbox.